module/zfs/zio_compress.c

```diff
@@ -166,3 +209,40 @@ zio_decompress_data(enum zio_compress c, abd_t *src, void *dst,
 
 	return (ret);
 }
+
+
+int
+zio_decompress_getcomplevel(enum zio_compress c, void *src, size_t s_len,
+    uint8_t *level)
+{
+	int ret;
+	zio_compress_info_t *ci = &zio_compress_table[c];
+
+	if ((uint_t)c >= ZIO_COMPRESS_FUNCTIONS || level == NULL)
+		return (SET_ERROR(EINVAL));
+
+	/* Not having this function is non-fatal */
+	if (ci->ci_get_level == NULL)
+		ret = SET_ERROR(EOPNOTSUPP);
+	else
+		ret = ci->ci_get_level(src, s_len, level);
+
+	return (ret);
+}
+
+int
+zio_compress_to_feature(enum zio_compress comp)
+{
+	/* XXX: Allan */
```

Comment:
This is junk: remove

Reply:
isnt it self explaining? consider that most of the function in zfs_ioctl are not documented